index.html

```diff
@@ -85,139 +71,160 @@ <h2>Актуальные предложения</h2>
         <li>
           <h3>ПЕРФОРАТОРЫ</h3>
           <p>Настоящие мужские игрушки</p>
+          <button type="button" name="buttonLeft"></button>
+          <button type="button" name="buttonRight"></button>
+          <form action="https://echo.htmlacademy.ru/courses" method="post">
+            <label><input type="radio" name="sliderIndicatorLeft" checked></label>
+            <label><input type="radio" name="sliderIndicatorRight"></label>
+          </form>
+          <a href="catalog.html">открыть каталог</a>
+        </li>
+        <li>
+          <h3>ДРЕЛИ</h3>
+          <p>Соседям на радость!</p>
+          <button type="button" name="buttonLeft"></button>
+          <button type="button" name="buttonRight"></button>
+          <form action="https://echo.htmlacademy.ru/courses" method="post">
+            <label><input type="radio" name="sliderIndicatorLeft"></label>
+            <label><input type="radio" name="sliderIndicatorRight" checked></label>
+          </form>
           <a href="catalog.html">открыть каталог</a>
         </li>
       </ul>
     </section>
     <section>
       <h2>популярные товары:</h2>
+      <p><a href="">все популярные товары</a></p>
       <ul>
         <li><a href="product.html">
-            <h3>Перфоратор BOSCH<br>BFG 9000</h3>
             <img src="img/product-1.jpg" width="184" height="164" alt="Перфоратор BOSCH BFG 9000">
-            <p>32500 Р.</p>
-            <p>25500 Р.</p>
+            <h3>Перфоратор BOSCH BFG 9000</h3>
+            <p><del>32500 Р.</del></p>
+            <p><ins>25500 Р.</ins></p>
           </a>
         </li>
         <li>
           <a href="product.html">
-            <h3>Перфоратор BOSCH<br>BFG 3000</h3>
             <img src="img/product-2.jpg" width="184" height="164" alt="Перфоратор BOSCH BFG 3000">
-            <p>22500 Р.</p>
-            <p>15500 Р.</p>
+            <h3>Перфоратор BOSCH BFG 3000</h3>
+            <p><del>22500 Р.</del></p>
+            <p><ins>15500 Р.</ins></p>
           </a>
         </li>
         <li>
           <a href="product.html">
-            <h3>Перфоратор BOSCH<br>BFG 6000</h3>
             <img src="img/product-3.jpg" width="184" height="164" alt="Перфоратор BOSCH BFG 6000">
-            <p>30500 Р.</p>
-            <p>25500 Р.</p>
+            <h3>Перфоратор BOSCH BFG 6000</h3>
+            <p><del>30500 Р.</del></p>
+            <p><ins>25500 Р.</ins></p>
           </a>
         </li>
         <li>
           <a href="product.html">
-            <h3>Перфоратор BOSCH<br>BFG 3000</h3>
             <img src="img/product-4.jpg" width="184" height="164" alt="Перфоратор BOSCH BFG 2000">
+            <h3>Перфоратор BOSCH BFG 3000</h3>
             <p>12500 Р.</p>
           </a>
         </li>
       </ul>
-      <a href="blank.html">все популярные товары</a>
+
     </section>
     <section>
       <h2>популярные производители:</h2>
+      <p><a href="">все производители</a></p>
```

Comment:
Тут нужно объединить в один блок, можно в `header` или `div`
```suggestion 
  <header>
      <h2>популярные производители:</h2>
      <a href="">все производители</a>
   </header>
```